### PR TITLE
Feature div exprt accessors

### PR DESCRIPTION
**Pull request by hannes-steffenhagen-diffblue**
- [x] Each commit message has a non-empty body, explaining why the change was made.
- [x] My contribution is formatted in line with CODING_STANDARD.md.
- [x] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [x] My commit message includes data points confirming performance improvements (if claimed). (**not applicable**)
- [x] My PR is restricted to a single feature or bugfix.
- [x] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: unit/util/expr.cpp [hunk 1]
```diff
@@ -37,3 +37,12 @@ SCENARIO("bitfield-expr-is-zero", "[core][util][expr]")
     }
   }
 }
+
+TEST_CASE("div_exprt accessors")
```
Comment (tautschnig) [inline unit/util/expr.cpp]: I think this should go in (a new file) `unit/util/std_expr.cpp`

**Review by tautschnig — APPROVED** [review]
Looks good except 1) the comment below (unit test in a different file) and 2) you might want to start using it across the code base?

File: unit/util/expr.cpp [hunk 2]
```diff
@@ -37,3 +37,12 @@ SCENARIO("bitfield-expr-is-zero", "[core][util][expr]")
...
+{
+  auto dividend = from_integer(10, integer_typet());
+  auto divisor = from_integer(5, integer_typet());
+  auto div = div_exprt(dividend, divisor);
+  REQUIRE(div.dividend() == dividend);
+  REQUIRE(div.divisor() == divisor);
+}
```
Comment (kroening) [inline unit/util/expr.cpp]: While at it, I would also check that the type of div is integer_typet().

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 71a470ddc6b8472fb0e07121ac73384996846f86).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85767105
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: b368af08ee1fcdeabcd3ddbcb085feb73d318c6f).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85768445
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: e4f63000a82853f785593eca93a3d4d552e563a9).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85769611

**Review by peterschrammel — APPROVED** [review]

**Comment by hannes-steffenhagen-diffblue** [comment]
@kroening @tautschnig 

I've addressed your comments as they pertain to the changes made here. As to using it elsewhere in the code base: That was the plan, but my preference would to update code to use these when we have to change it anyway, similar to how we've done other code style changes, rather than update it across the code base in bulk in this PR.

**Comment by tautschnig** [comment]
@hannes-steffenhagen-diffblue CI appears to be failing.

**Comment by hannes-steffenhagen-diffblue** [comment]
@tautschnig I'm looking into it

Got it, just forgot to add the new file to the unit test makefile

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: de878424226bcf44e88b96b76642c5098b0d3f4b).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/85887732

**Pull request merged**